SPECS/xcp-python-libs.spec

```diff
@@ -1,6 +1,6 @@
 %global package_speccommit 4667fd527d3183fdb1968f2be7041a5a0ebe927c
 %global usver 3.0.2
-%global xsver 4
+%global xsver 4.1
```

Comment:
You can set it to 4.2 and update the release in the changelog. We'll build it right after the merge.

Reply:
done